basic-example/Dockerfile.jupyterhub

```diff
@@ -4,9 +4,9 @@ ARG JUPYTERHUB_VERSION
 FROM jupyterhub/jupyterhub:$JUPYTERHUB_VERSION
 
 # Install dockerspawner, nativeauthenticator
+# hadolint ignore=DL3013
 RUN python3 -m pip install --no-cache-dir \
-        dockerspawner==12.* \
-        jupyterhub-nativeauthenticator==1.*
+    dockerspawner \
+    jupyterhub-nativeauthenticator
 
-COPY ./jupyterhub_config.py /srv/jupyterhub/jupyterhub_config.py
```

Comment:
I was wrong here, this file is being mounted as part of docker-compose as well.
So, copy here makes no sense.